pkg/translator/prometheusremotewrite/helper.go

```diff
@@ -26,7 +26,6 @@ import (
 )
 
 const (
-	nameStr       = "__name__"
```

Comment:
Why was this removed?

Reply:
It's because it was duplicating `github.com/prometheus/common/model.MetricNameLabel`. `model.MetricNameLabel` is already used elsewhere in this package, so this change also ensures consistency.